Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:20-alpine AS deps
+RUN apk add --no-cache libc6-compat openssl
+WORKDIR /app
+COPY package.json yarn.lock ./
+RUN yarn --frozen-lockfile
+
+FROM node:20-alpine AS builder
+ARG APP_FILE_STORAGE_DIR
+ARG APP_PUBLIC_URL
+WORKDIR /app
+COPY --from=deps /app/node_modules ./node_modules
+COPY . .
+
+RUN SKIP_ENV_VALIDATION=1 yarn build
+
+FROM gcr.io/distroless/nodejs20-debian12 AS runner
+WORKDIR /app
+
+ENV NODE_ENV production
+
+COPY --from=builder /app/next.config.mjs ./
```

Comment:
Jedino što je potrebno kopirati su `public`, `.next/standalone` i `.next/static`, pls makni ostale